buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -87,27 +84,27 @@ object Versions {
 
   const val rxandroid: String = "2.1.0" // available: "2.1.1"
 
-  const val core_ktx: String = "1.2.0"
+  const val core_ktx: String = "1.2.0" // available: "1.3.0"
 
   const val kiwixlib: String = "9.1.2"
 
   const val material: String = "1.1.0-beta02" // available: "1.1.0"
 
   const val multidex: String = "2.0.1"
 
-  const val barista: String = "2.7.1" // available: "3.2.0"
+  const val barista: String = "2.7.1" // available: "3.5.0"
 
   const val xfetch2: String = "3.1.4"
 
   const val jsr305: String = "3.0.2"
 
-  const val ktlint: String = "0.36.0"
+  const val ktlint: String = "0.36.0" // available: "0.37.1"
```

Comment:
Minor versions can be updated.

Reply:
I'd prefer to do that in a separate update, upgrading ktlint could introduce extra noise and this is already a good deal noisier than I like because I upgraded to AS 4.0.0 and that enforced new lint rules, hence the `requireXXX()`s